packages/trader/src/App/Components/Elements/VideoStream/video-controls.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Text, Icon } from '@deriv/components';
+import VolumeControl from './volume-control';
+import PlaybackRateControl from './playback-rate-control';
+
+type TVideoControls = {
+    block_controls?: boolean;
+    current_time?: number;
+    is_playing?: boolean;
+    is_mobile?: boolean;
+    onRewind: (e: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+    dragStartHandler: (e: React.MouseEvent<HTMLElement, MouseEvent> | React.TouchEvent<HTMLElement>) => void;
+    onVolumeChange: (new_value: number) => void;
+    onPlaybackRateChange: (new_value: number) => void;
+    togglePlaying: (e: React.MouseEvent<HTMLElement, MouseEvent>) => void;
+    video_duration?: number;
+    show_controls?: boolean;
+    volume?: number;
+};
+
+const VideoControls = React.forwardRef<HTMLDivElement, TVideoControls>(
+    (
+        {
+            block_controls,
+            current_time,
+            is_playing,
+            is_mobile,
+            onRewind,
+            dragStartHandler,
+            onVolumeChange,
+            onPlaybackRateChange,
+            togglePlaying,
+            show_controls,
+            video_duration,
+            volume,
+        },
+        ref
+    ) => {
+        const [is_drag_dot_visible, setIsDragDotVisible] = React.useState(false);
+
+        const dummy_time = '00:00';
+
+        const formatTime = (time?: number) => {
```

Comment:
I can move to the shared package (to the file with functions helpers, which are working with time)